README.md

```diff
@@ -15,6 +15,214 @@ Build
 
     $ npm run dist
 
+Usage
+-----
+
+**Caution: everything in this section is still pure fiction.**
+
+* Every operation is performed locally;
+* Synchronization with server shall be ran explicitly.
+
+### Collection
+
+```js
+
+  import Cliquetis from "cliquet";
+
+  var db = new Cliquetis();
+  var articles = db.collection('articles');
+
+```
+
+The collection object has the following attributes:
+
+* **last_modified**: last synchronization timestamp, ``undefined`` if never sync'ed.
+
+> Synchronization timestamps are persisted in the device local storage.
+
+
+### List records
+
+```js
+
+  articles.all().then(result => {
+    {
+      data: [
+        {
+          id: "705b17be-e957-4c14-8f4c-86f8eaac29c0",
+          title: "foo"
+        },
+        {
+          id: "68e63131-3859-40cc-a4f7-b237ca179329",
+          last_modified: 1432222889336,
+          title: "Web page"
+        },
+      ]
+    }
+  });
+
+```
+
+> Records with ``last_modified`` attribute were sync'ed on a server.
+
+
+### Create records
+
+```js
+
+  articles.save({title: "foo"}).then(result => {
+    data: {
+      id: "2dcd0e65-468c-4655-8015-30c8b3a1c8f8",
+      title: "foo",
+    }
+  });
```

Comment:
Note that if we want to keep this formalism, we should at least make the code valid, wrapping the returning object into parenthesis:

```
articles.save({title: "foo"}).then(result => ({
  data: {
    id: "2dcd0e65-468c-4655-8015-30c8b3a1c8f8",
    title: "foo",
  }
}));
```